addons/hw_posbox_homepage/controllers/homepage.py

```diff
@@ -46,7 +46,7 @@ def __init__(self):
         super().__init__()
         self.updating = threading.Lock()
 
-    @route.iot_route('/', type='http')
+    @route.iot_route(['/hw_posbox_homepage'] + (['/'] if not iot_system.IS_IOT_TEST else []), type='http')
```

Comment:
```suggestion
    @route.iot_route('/hw_posbox_homepage' if iot_system.IS_IOT_TEST else '/'), type='http')
```
Could be simplified, do we really need both endpoint if the iot_system is test?